server/api.js

```diff
@@ -63,4 +59,32 @@ router.get("/pages/:title", async (req, res) => {
 	}
 });
 
+
+
+// Deleting module endpoint
+router.delete("/pages/:page_title/:record_id", async (req, res) => {
+	try {
+		const pageTitle = req.params.page_title;
+
+		const record_id = req.params.record_id;
+
+		const findPageId = await db.query(
+			`select page_id from pages where page_title = '$1'`,[pageTitle]
```

Comment:
This query is giving me error. because you have
`select page_id from pages where page_title = '$1'`

while it should be `select page_id from pages where page_title = $1`

Reply:
done